packages/server/src/api/controllers/table/external.ts

```diff
@@ -16,6 +16,7 @@ import {
 import sdk from "../../../sdk"
 import { builderSocket } from "../../../websockets"
 import { inputProcessing } from "../../../utilities/rowProcessor"
+import _ from "lodash"
```

Comment:
usually preferred to use `import { isEqual } from "lodash"` as it allows just that function to be pulled in here as part of the esbuild.